sdk/node/README.md

```diff
@@ -1,266 +1,20 @@
-#Hyperledger Fabric Client SDK for Node.js
+## Hyperledger Fabric Client SDK for Node.js
 
 The Hyperledger Fabric Client SDK (HFC) provides a powerful and easy to use API to interact with a Hyperledger Fabric blockchain.
 
-To install `hfc` from npm simply execute the following command:
+To learn about how to install and use the Node.js SDK, please visit the [fabric documentation](http://hyperledger-fabric.readthedocs.io/en/latest/Setup/NodeSDK-setup).
 
-    npm install -g hfc
+The [Going Deeper](#going-deeper) section discusses HFC's pluggability or extensibility design.  It also describes the main object hierarchy to help you get started in navigating the reference documentation. The top-level class is `Chain`.
```

Comment:
"pluggability or extensibility design" -> pluggability and extensibility design

Should be one space after periods instead of two.